### ISSUE TITLE
Go to build task goes to empty form

### ISSUE THREAD

**Issue by deboer-tim** [containers/podman-desktop#10079]
### Bug description

Once a build is complete (successfully or after an error), clicking on the 'Go to task' link goes to a blank build form.

I would have expected it to still go to the build so that I can see the full output or the error message in context. If we can't go back, the link should probably be removed?

### Operating system

macOS 15.1

### Installation Method

None

### Version

1.14.1

### Relevant log output

```shell

```

**Comment by odockal** [comment]
@deboer-tim I think this is a dupe of https://github.com/podman-desktop/podman-desktop/issues/6062. Basically getting back to any task after it is done show blank/default form

**Comment by deboer-tim** [comment]
Agreed it's the same problem, but I'm guessing that neither page implements the goto link correctly and will need to be fixed independently.
